### DISCUSSION TITLE
Which url to blacklist for jupyter lab?

### DISCUSSION THREAD

**Discussion by jpbohl** [tridactyl/tridactyl#4237] [Q&A]
Hi, I want to blacklist Jupyter Lab in tridactyl so that I can use their key bindings, however I am not sure which URL I have to blacklist for this purpose. My Jupyter instances usually run on URLs such as _http://localhost:8888/lab/_. Any help is greatly appreciated.

**Comment by bovine3dom** [comment]
It's just a regex so e.g. `:blacklistadd localhost:[0-9]+\/lab` if it's always on localhost but the port sometimes changes :)
